### ISSUE TITLE
Automatic clon

### ISSUE THREAD

**Issue by eXponenta** [englercj/playground#10]
Automatic clone when trying edit  another's fiddle.

**Comment by bigtimebuddy** [comment]
I think the more important feature here is to accept a github login, like jsfiddle/codepen do, so that playgrounds are readonly/auto-clone unless it's your's.

**Comment by englercj** [comment]
Yup been wanting to add this feature for a while, just haven't had time to get to it.

**Comment by bigtimebuddy** [comment]
Some basic info, when you're ready to implement:
https://developer.github.com/v3/guides/basics-of-authentication/
